src/main/java/novamachina/exnihilosequentia/common/compat/kubejs/HeatRecipeJS.java

```diff
@@ -0,0 +1,38 @@
+package novamachina.exnihilosequentia.common.compat.kubejs;
+
+import com.google.gson.JsonObject;
+import dev.latvian.mods.kubejs.item.ItemStackJS;
+import dev.latvian.mods.kubejs.util.ListJS;
+import dev.latvian.mods.kubejs.util.MapJS;
+
+public class HeatRecipeJS extends BaseRecipeJS {
+
+  @Override
+  public void create(ListJS args) {
+    this.source(ItemStackJS.of(args.get(0)));
+    this.amount(((Number) args.get(1)).intValue());
+    this.properties((MapJS)args.get(2));
```

Comment:
i first had trouble making a heat recipe, but after adding `{}` as empty additional properties, it works

Reply:
I'll make this an optional parameter and then make sure to add this to the documentation.